### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by MahmoudSayedA**
from mahmoudSA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Chatting App
```
Comment (mah13-S) [inline README.md:1]: where is the rest?

Comment (mah13-S) [inline README.md:1]: it's good ?

Comment (MahmoudSayedA) [reply README.md:1]: it's just  a startup to go

Comment (MahmoudSayedA) [reply README.md:1]: ya it's nice

**Pull request merged**